modules/Library/library_manage_catalog_editProcess.php

```diff
@@ -83,13 +81,10 @@
             } else {
                 $replacement == 'N';
             }
-            $comment = $_POST['comment'];
-            $gibbonSpaceID = null;
-            if ($_POST['gibbonSpaceID'] != '') {
-                $gibbonSpaceID = $_POST['gibbonSpaceID'];
-            }
-            $locationDetail = $_POST['locationDetail'];
-            $ownershipType = $_POST['ownershipType'];
+            $comment = $_POST['comment'] ?? '';
+            $gibbonSpaceID = $_POST['gibbonSpaceID'] ?? null;
```

Comment:
@SKuipers would probably like it better if the fallback is '' instead of `null`.